arch/arm64/boot/dts/qcom/sm7150-xiaomi-tucana.dts

```diff
@@ -85,23 +102,81 @@
 	clock-frequency = <100000>;
 	status = "okay";
 
-	/* ti,drv2605l (haptic) @ 5a */
-	/* awinic,aw3644 (flash led) @ 63 */
+	haptics@5a {
+		compatible = "ti,drv2605l";
+		reg = <0x5a>;
+		enable-gpios = <&tlmm 4 GPIO_ACTIVE_HIGH>;
+		mode = <DRV260X_LRA_NO_CAL_MODE>;
+		library-sel = <DRV260X_LIB_LRA>;
+		vib-rated-mv = <3200>;
+		vib-overdrive-mv = <3200>;
+	};
+
+	softlight: aw3644@63 {
+		compatible = "awinic,aw3644";
+		reg = <0x63>;
+		aw3644,hwen-gpio = <&pm6150l_gpios 12 GPIO_ACTIVE_HIGH>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		led@0 {
+			reg = <0>;
+			function = LED_FUNCTION_FLASH;
+			function-enumerator = <2>;
+			color = <LED_COLOR_ID_WHITE>;
+		};
+
+		led@1 {
+			reg = <1>;
+			function = LED_FUNCTION_FLASH;
+			function-enumerator = <3>;
+			color = <LED_COLOR_ID_WHITE>;
+		};
+	};
 };
 
 &i2c7 {
 	clock-frequency = <100000>;
 	status = "okay";
 
-	/* st,fts (touchscreen) @ 49 */
+	touchscreen: fts@49 {
+		compatible = "st,fts";
+		reg = <0x49>;
+		interrupt-parent = <&tlmm>;
+		interrupts = <9 0x2008>;
+		pinctrl-0 = <&ts_int_active &ts_reset_active>;
+		pinctrl-1 = <&ts_int_suspend &ts_reset_suspend>;
+		pinctrl-names = "default", "sleep";
+		vdd-supply = <&vreg_touch_vdd>;
+		avdd-supply = <&vreg_l6c_3p0>;
+		fts,regulator_dvdd = "vdd";
+		fts,regulator_avdd = "avdd";
+		fts,irq-gpio = <&tlmm 9 0x2008>;
+		fts,reset-gpio-enable;
+		fts,reset-gpio = <&tlmm 8 GPIO_ACTIVE_HIGH>;
+		fts,irq-flags = <0x2008>; /* IRQF_ONESHOT | IRQF_TRIGGER_LOW */
```

Comment:
If you're going to comment the flags here, just use them

Reply:
Those flags aren't present in include/dt-bindings, so should I just remove the comment?